tests/std/tests/P0355R7_calendars_and_time_zones_time_zones/test.cpp

```diff
@@ -91,9 +117,268 @@ void timezone_names_test() {
     assert(_Locate_zone_impl(tzdb.zones, "America/Nuuk") == nullptr); // does not exist in ICU (very rare)
 }
 
+void validate_timezone_transitions(const time_zone* tz, const Transition& data) {

```

Comment:
`data` is the name of a function in `std`; I recommend choosing any other name to avoid "shadowing" even if there are no shadow warnings.